test/dummy/app/tasks/maintenance/params_task.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module Maintenance
+  class ParamsTask < MaintenanceTasks::Task
+    attribute :post_ids, :string
+
+    validates :post_ids, presence: true
+    validates_format_of :post_ids, with: /\A(\s?\d+(,\s?\d+\s?)*)\z/,
+      allow_blank: true
```

Comment:
Maybe we could combine them?
```suggestion
    validates :post_ids,
      presence: true,
      format: { with: /\A(\s?\d+(,\s?\d+\s?)*)\z/, allow_blank: true }
```

Reply:
Sure, this is going to change with https://github.com/Shopify/maintenance_tasks/pull/414 though 😉